library/agent/api/Event.ts

```diff
@@ -82,7 +94,7 @@ type MonitoredSinkStats = {
 type Heartbeat = {
   type: "heartbeat";
   stats: {
-    sinks: Record<string, MonitoredSinkStats>;
+    operations: OperationStats[];
```

Comment:
would still provide a key no? interesting data

Reply:
previously we used the sink name (e.g. mongodb) as key for the object

I moved the `kind` inside the array now, I think it makes it easier to change the format later on 😬